controller/index.js

```diff
@@ -1,39 +1,51 @@
 const model = require('../model/index.js');
 
 module.exports = {
-  post: {
-    productImages: async (req, res) => {
+  productImages: {
+    post: async (req, res) => {
       let { shoeId } = req.params;
-      console.log('req:', req.body.imgUrl);
       let { imgUrl } = req.body;
-      console.log(`shoeid ${shoeId}, imgurl ${imgUrl}`)
       try {
-        const imgEntry = await model.post.shoeImgs(shoeId, imgUrl);
-        res.status(200).send(imgEntry);
+        const postResponse = await model.mySql.shoeImgs.post(shoeId, imgUrl);
+        res.status(200).send(postResponse);
       } catch(err) {
-        console.log(`error in model POST ${err}`);
-        res.status(400);
+        console.log(`error in adding entry to DB ${err}`);
```

Comment:
you may want to send these error messages in your responses instead of just console logging them